tests/oauth2/test_service_account.py

```diff
@@ -58,15 +58,14 @@ def credentials_fixture(self, signer):
             signer, self.SERVICE_ACCOUNT_EMAIL, self.TOKEN_URI)
 
     def test_from_service_account_info(self):
-        with open(SERVICE_ACCOUNT_JSON_FILE, 'r') as fh:
-            info = json.load(fh)
```

Comment:
Pete and re-pete were on a boat?

You seem to like `info = SERVICE_ACCOUNT_INFO.copy()` everywhere else (would've also been a smaller diff, "diff-driven development")